pkg/quota/install/registry.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
That is unusual usage of install subdir. Looking through current source dirs, I'd rather to leave install subdir for installing specific API group. Looking at this code all it does is just forward call to quota internal pkg, is there some deeper intention here I'm missing here?